### PR TITLE
add endpoint cities list with location

### PR DESCRIPTION
**Pull request by ayocodingit**
# overview

- add endpoint cities list with location

the goal is bounds to get a city marker

cc: @jabardigitalservice/jds-backend 

## Evidence
- title: add endpoint cities list with location
- project: Desa Digital
- participants: @ayocodingit @firmanJS @setiadijoe

### CODE REVIEW

File: src/modules/cities/city_service.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        },
+      })
+    }
+
+    return data
+  }
+
+  export const findAllWithLocation = async (requestQuery: Entity.RequestQuery): Promise<Entity.ResponseFindAllWithLocation> => {
+    const items = await Repository.findAllWithLocation(requestQuery)
+
```
Comment (firmanJS) [inline src/modules/cities/city_service.ts:23]: Ini harusnya await repository

Bukan await meta.toal

File: src/modules/cities/city_service.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    const meta: any = Repository.metaFindAllWithLocation()
+    const total: any = await meta.total
```
Comment (firmanJS) [inline src/modules/cities/city_service.ts:25]: ini harusnya awaitnya di repository bukan i meta.toal

File: src/modules/cities/city_service.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+    const result: Entity.ResponseFindAllWithLocation = {
+      data: responseFindAllWithLocation(items),
+      meta: {
+        total: total.total,
```
Comment (setiadijoe) [inline src/modules/cities/city_service.ts:30]: ini jadinya redundant sama line 25, kenapa gak langsung aja di input di result?

File: src/modules/cities/city_service.ts (hunk 2 above)
Comment (ayocodingit) [reply src/modules/cities/city_service.ts]: update mangs 61c2062

**Review by firmanJS — APPROVED** [review]

**Review by tukangremot — APPROVED** [review]

**Review by setiadijoe — APPROVED** [review]

**Review by hafizulf — APPROVED** [review]

**Pull request merged**